code.js

```diff
@@ -1,3 +1,39 @@
-function hasCycle(graph) {
+function hasCycle(graph, start = 0, visited = []) 
```

Comment:
What if there's a cycle starting at a different node?

Reply:
It can work for any start node as long as the edges go both ways in the graph right?

Reply:
So are you saying that your code works regardless of the start node or that I would have to run it multiple times for different start nodes?